app/models/account.rb

```diff
@@ -5,14 +5,15 @@ class Account < ApplicationRecord
          :recoverable, :rememberable, :validatable, :confirmable, :lockable
 
   has_many :subscriptions
-  has_many :communities, through: :subscriptions
+  has_many :communities
```

Comment:
Are you sure you want to remove the through relationship?

Reply:
No magesh i tried  through relationship for testing purpose i will restore it.